### ISSUE TITLE
removeChangeSetProperty does not work with included changelogs

### ISSUE THREAD

**Issue by szape89** [liquibase/liquibase#5290]
### Search first

- [X] I searched and no similar issues were found

### Description

There was an [issue](https://github.com/liquibase/liquibase/issues/3091) introduced a long time ago related to `afterColumn`, which is supposed to be [fixed](https://github.com/liquibase/liquibase/pull/3978) by introducing `removeChangeSetProperty`.
I [asked](https://github.com/liquibase/liquibase/issues/3091#issuecomment-1816524161) at the original issue if this should work with included changelogs, but I did not get a response, so here I am.
In case of XML format I can only get it working if inside `databaseChangeLog` the `removeChangeSetProperty` is followed by multiple `changeSet` but not with multiple `include` or a single `includeAll`.
I'm not sure if this is intentional or this scenario was forgotten, but regardless its a reasonable expectation that this should work.

### Steps To Reproduce

Use the following snippet for example to reproduce:

```
<databaseChangeLog
	xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
		http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
	<removeChangeSetProperty change="addColumn" dbms="mssql" remove="afterColumn" />
	<includeAll path="db/changelog/schemas" />
</databaseChangeLog>
```

### Expected/Desired Behavior

`removeChangeSetProperty` should work with included changelogs too.

### Liquibase Version

4.25.0

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by tati-qalified** [comment]
Hi @szape89, I see where you're coming from, and I agree that it would be useful for the `removeChangeSetProperty` statement to work for included changelogs.

This use case was not considered when the fix was implemented, so right now it will not work. Would you be willing to create a PR for it? In that case, we would provide you with guidance when needed.

Thank you,
Tatiana

**Comment by php-coder** [comment]
Yes, this is pretty much expected behavior because otherwise we have to manually modify a lot of old changelogs :( I'm not happy with the fact that Liquibase isn't backward compatible but if it breaks it's better to have a small fix.

**Comment by tati-qalified** [comment]
Hi @php-coder 
The attribute was created by a community member and not by us, so backwards compatibility is not guaranteed. If you would benefit from `removeChangeSetProperty` also working for included files, you could create a PR and we could guide you through the writing process.

In the meantime, we'll leave the ticket open for anyone in the community that is interested in fixing it.
Thank you,
Tatiana

**Comment by szape89** [comment]
Hi @tati-qalified, sorry for the late reply, I was out office for the holidays but wanted to consider possible options before answering. For now I chose a workaround, because based on the history of this issue I do not know when it will be resolved, if at all, but I had to get on with my work.

For anybody interested, removing the "invalid" attribute (e.g. `afterColumn`) and using `validCheckSum` on the changeset was my fix.

Regarding a real solution it would be useful to point in the direction where the issue might be for whoever takes on this task. Having no internal knowledge of the library my best guess is that the previous [changes](https://github.com/liquibase/liquibase/pull/3978/files) are not applied to included changesets for some reason.
